news/2020-19-11-mappers-report-september-october.md

```diff
@@ -0,0 +1,211 @@
+
+For MBC #8 the restriction was that sliders could not be used, and [Mao](https://osu.ppy.sh/users/2204515)'s entry of [Silentroom - NULCTRL](https://osu.ppy.sh/beatmapsets/1266008#osu/2631203) with its excellent use of circles, **circle size 7**, and screen-usage based patterning won over the judge's hearts. He has now earned enough points in MBC for a tier 2 profile badge, and is all that much closer to attaining the coveted title of elite mapper! Well deserved!
+
+In MBC #9 the theme was that every object must overlap visible with another one. With its results mapper [xbopost](https://osu.ppy.sh/users/6842421) came in first place with his entry of [PUP - See You At Your Funeral](https://osu.ppy.sh/beatmapsets/1289138#osu/2676185) which took the theme well to heart and built a unique and interesting map and gameplay experience that you should be sure to check out! That's not all though, as this is xboposts' **third** time coming in first place in MBC and they have now earned the esteemed title of **Elite Mapper**! It's certainly well deserved, with each of their entries providing well built and interesting experiences that make excellent use of the challenges MBC provides. They are certainly a mapper to keep up with in the future as well!
+
+### osu! Beatmapping World Championship 2020
+
+After 5 fast months, the osu! Beatmapping World Championship 2020 has come to an end. It's difficult to express just how amazing of a project and competition it was to watch, and each team brought their best selves - resulting in some of the closest rounds of the competition right at the end! For full details on the journey, scoring, and more, be sure to check out the [dedicated conclusion newspost](https://osu.ppy.sh/home/news/2020-10-22-beatmapping-world-championship-2020-concludes).
+
+The final results, in a simplified form are as follows:
+
+#### Finals
+
+| Team 1 | | | Team 2 |
+| --: | :-: | :-: | :-- |
+| ![](/wiki/shared/flag/AR.gif) Argentina | 3 | **4** | **United Kingdom** ![](/wiki/shared/flag/GB.gif) |
+
+#### Third Place
+
+| Team 1 | | | Team 2 |
+| --: | :-: | :-: | :-- |
+| ![](/wiki/shared/flag/BR.gif) **Brazil** | **4** | 3 | Russian Federation ![](/wiki/shared/flag/RU.gif) |
+
+Congratulations to everyone!
+
+## <a id="around-community"></a>Around the Community
+
+Hello! We're back with the latest segment of around the community! This time we have two main spotlights: the return of the annual christmas queue, as well as a bunch of awesome guides that have been put out recently! Should you have something worth including that isn't here already, be sure to let [Feerum](https://osu.ppy.sh/users/4815717) or [Noffy](https://osu.ppy.sh/users/1541323) know about it! We're happy to give it more attention in our news post!
```

Comment:
```suggestion
Hello! We're back with the latest segment of Around the Community! This time we have two main spotlights: the return of the annual Christmas Queue, as well as a bunch of awesome guides that have been put out recently! Should you have something worth including that isn't here already, be sure to let [Feerum](https://osu.ppy.sh/users/4815717) or [Noffy](https://osu.ppy.sh/users/1541323) know about it! We're happy to give it more attention in our news post!
```
'Around the Community' sounds like a proper noun, hence the capitalization
the same thing can also be considered for the 'annual Christmas Queue'